lib/remap-functions/src/encode_base64.rs

```diff
@@ -43,7 +43,7 @@ impl Function for EncodeBase64 {
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-enum Charset {
+pub enum Charset {
```

Comment:
I think it'd be better to move the base64 remap logic into a submodule: `base64` where this charset could be defined in `mod.rs`, given that it is shared, and the functions could live along side in `base64/encode.rs` and `base64/decode.rs`.